packages/react-inbox/src/components/Message/styled.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { useMemo } from "react";
 import styled from "styled-components";
-import CourierSvg from "./courier-icon.svg";
+import CourierSvg from "../../assets/courier-icon.svg";
```

Comment:
you should be able to do ~/assets

Reply:
There was an issue I encountered with having this and `babel-plugin-root-import`. Fixed.